src/Jackett.Common/Definitions/rutor.yml

```diff
@@ -70,6 +74,11 @@ search:
   fields:
     category:
       text: 3
+    title_is_rus:
+      selector: a[href^="viewtopic.php?t="] > b
+      filters:
```

Comment:
you need to use the selector for the title as below, this is not nonameclub ;-)

Reply:
Well that would just be sensible... 🤦 brb